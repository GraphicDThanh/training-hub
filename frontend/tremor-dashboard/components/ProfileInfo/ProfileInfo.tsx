import { memo } from "react";

// Components
import { Text } from "@tremor/react";
import { Avatar } from "@/components";

// Constants
import { USER_ROLE } from "@/types";

interface ProfileInfoProps {
  name: string;
  role: USER_ROLE | string;
  avatarUrl: string;
  isProject?: boolean;
}

const ProfileInfo = ({
  name,
  role,
  avatarUrl,
  isProject,
}: ProfileInfoProps) => {
  return (
    <div className="flex flex-col lg:flex-row lg:items-center justify-between">
      <div
        className={`flex flex-col xs:flex-row xs:items-center flex-wrap profile-info ${
          isProject ? "my-1" : "mb-3 -mt-3 -ml-2"
        }`}>
        <Avatar
          alt={`${name} avatar`}
          className="shadow-lg w-[50px] h-[50px]"
          height={50}
          priority
          src={avatarUrl}
          width={50}
        />
        <div className="py-4 ml-6" data-testid="profile-info-lg">
          <h2 className="font-semibold text-xl text-primary dark:text-dark-primary capitalize">
            {name}
          </h2>
          <Text className="font-normal text-tertiary dark:text-dark-romance">
            {role}
          </Text>
        </div>
      </div>
    </div>
  );
};

export default memo(ProfileInfo);
