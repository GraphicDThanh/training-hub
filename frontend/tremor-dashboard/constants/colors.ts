type ColorChannelsChartType = {
  [key: number]: string;
  1: string;
  2: string;
  3: string;
  4: string;
};

export const COLOR_CHANNELS_CHART: ColorChannelsChartType = {
  1: "blue-500",
  2: "red-500",
  3: "primary",
  4: "gray-500",
};
