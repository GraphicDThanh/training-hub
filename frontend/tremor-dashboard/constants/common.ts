// Types
import { OptionType } from "@/types";

export const EXCEPT_KEYS = {
  POSITIVE_INTEGER: ["e", "E", "+", "-", "."],
  POSITIVE_DOUBLE: ["e", "E", "+", "-"],
};

export enum DIRECTION {
  ASC = "asc",
  DESC = "desc",
}

// Mailto Support
export const MAILTO_SUPPORT = "support@tremor-dashboard.com";

export const REMEMBER_ME_COOKIES_KEY = "remember-me";
export const UID_KEY = "uid";
export const ROLE = "role";
export const NAME = "name";
export const AVATAR = "avatar";
export const AUTH_SESSION_COOKIES_KEY = "authjs.session-token";

export const ORDER_LIST_OPTIONS: OptionType[] = [
  { option: "Paid", value: "0" },
  { option: "Refunded", value: "2" },
  { option: "Canceled", value: "1" },
];

export const PRODUCT_LIST_OPTIONS: OptionType[] = [
  { option: "Yes", value: "true" },
  { option: "No", value: "false" },
];

export const FORMAT_DATE = {
  YEAR_MONTH_DATE_TIME: "YYYY-MM-DDTHH:mm:ss.SSS",
  MONTH_DAY_YEAR_TIME: "MMM D, YYYY h:mm A",
};

export const SEPARATOR = {
  COMMAS: ",",
  DOTS: ".",
};

export const CDN_KEY = process.env.NEXT_PUBLIC_CDN_KEY;

export const DRAG_ZONE = {
  DEFAULT: {
    TEXT: "Drag and drop your file here to begin uploading.",
    STYLE_INPUT: "border-gray-300",
    STYLE_TEXT: "text-gray-500 dark:text-gray-400",
  },
  ON_DRAG: {
    TEXT: "Release the mouse click to upload the photo",
    STYLE_INPUT: "outline outline-2 outline-[#ebeff4]",
    STYLE_TEXT: "text-gray-500 dark:text-gray-400",
  },
  ON_WRONG_FORMAT: {
    TEXT: "Invalid image file type. Please try uploading again.",
    STYLE_INPUT: "border-red-500",
    STYLE_TEXT: "text-red-500 dark:text-red-500",
  },
};

export const TRANSACTION_CLASS = {
  INCREASE: {
    buttonClasses: "text-few dark:text-few dark:hove:text-few border-few",
    textClasses: "font-semibold text-greener dark:text-few",
  },
  DECREASE: {
    buttonClasses:
      "text-attention dark:text-attention dark:hover:text-attention border-attention",
    textClasses: "font-semibold text-blood dark:text-pink",
  },
  PENDING: {
    buttonClasses:
      "text-primary dark:text-primary dark:hover:text-primary border-primary",
    textClasses:
      "font-semibold text-primary dark:text-dark-tremor-content-pending",
  },
};

export const enum VARIANT_BUTTON {
  LIGHT = "light",
  LIGHT_CARD = "lightCard",
  LIGHT_CENTER = "lightCenter",
  LIGHT_STATUS = "lightStatus",
  SECONDARY = "secondary",
  SECONDARY_SHADOW = "secondaryShadow",
  PRIMARY = "primary",
  PRIMARY_CENTER = "primaryCenter",
  SURFACE = "surface",
  DARK = "dark",
  ACTIVE = "active",
}

export const NO_IMAGE = "No Image";

export const RESULT_NOT_FOUND = "Result Not Found";

export const AMOUNT_LIMIT = 20000;
export const PRICE_LIMIT = 20000;
export const QUANTITY_LIMIT = 10000;
export const WEIGHT_LIMIT = 2000;

export const PREFIX_PHONE_NUMBER = "+1";

export const STORAGES = {
  IS_DARK_THEME: "isDarkTheme",
};

export const THEMES = {
  LIGHT: "light",
  DARK: "dark",
};

export const LOGO_TITLE = "Tremor Dashboard PRO";
