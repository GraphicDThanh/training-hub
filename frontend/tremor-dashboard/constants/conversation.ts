export const CONVERSATION_TABLES = {
  GROUPS: "groups",
  CHATS: "chats",
};

export const CONVERSATION_ORDERS = {
  CREATED_AT: "createdAt",
  UPDATED_AT: "updatedAt",
};

export const CONVERSATION_QUERYS = {
  USER_IDS: "userIds",
};

export enum CONVERSATION_STATUS {
  INITIAL = "initial",
  FETCHED = "fetched",
}

export const ADMIN_ID = parseInt(process.env.NEXT_PUBLIC_ADMIN_ID!);

export enum DOCUMENT_CHANGE_TYPE {
  ADDED = "added",
  MODIFIED = "modified",
  REMOVED = "removed",
}

export enum WHERE_FILTER_OP {
  ARRAY_CONTAINS = "array-contains",
}
