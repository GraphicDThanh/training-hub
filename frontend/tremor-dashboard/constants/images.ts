export const NOT_FOUND_IMAGE = "/images/no-image-square.webp";
export const LOGO_IMAGE = "/assets/images/sidebar-logo.webp";
export const PLACEHOLDER_IMAGE = "/images/image-regular.svg";

export const AVATAR_IMAGE = {
  XS: "/images/avatar/avatar-xs.webp",
  SM: "/images/avatar/avatar-sm.webp",
  MD: "/images/avatar/avatar-md.webp",
  LG: "/images/avatar/avatar-lg.webp",
};

export const METADATA_IMAGE = {
  HOME_PREVIEW: "/assets/images/home-preview.webp",
  MASTERCARD: "/assets/images/mastercard.webp",
};

export const PROJECT_IMAGE = "/assets/images/profile-project.webp";
