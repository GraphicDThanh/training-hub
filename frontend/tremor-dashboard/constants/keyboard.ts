export const KEYBOARD_KEYS = {
  BACKSPACE: "Backspace",
  DIGIT_1: "1",
};
