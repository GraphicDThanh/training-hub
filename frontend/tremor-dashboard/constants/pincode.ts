export const PIN_CODE_DEFAULT = "*";
export const PIN_CODE_LENGTH = 6;
export const PIN_CODE_SKIP_SET_KEY = "skipSetupPinCode";
