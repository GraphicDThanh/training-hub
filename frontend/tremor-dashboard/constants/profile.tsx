import { PlatformSetting } from "@/types";

export const SETTING_FIELDS_ACCOUNT: PlatformSetting[] = [
  { label: "Email me when someone follows me", field: "emailMentions" },
  {
    label: "Email me when someone answers on my post",
    field: "emailFollowing",
  },
  { label: "Email me when someone mentions me", field: "emailAnswerPost" },
];

export const SETTING_FIELDS_APPLICATION: PlatformSetting[] = [
  { label: "New launches and projects", field: "newLaunchesProject" },
  { label: "Monthly product updates", field: "monthlyProductUpdate" },
  { label: "Subscribe to newsletter", field: "subscribeToNewsletter" },
];

export const SETTING_DATA_ACCOUNT = {
  emailMentions: false,
  emailFollowing: false,
  emailAnswerPost: false,
};

export const SETTING_DATA_APPLICATION = {
  newLaunchesProject: false,
  monthlyProductUpdate: false,
  subscribeToNewsletter: false,
};

// Social links
export const SOCIAL_LINKS = {
  facebook: "https://www.facebook.com/",
  twitter: "https://twitter.com/",
  instagram: "https://instagram.com",
};
