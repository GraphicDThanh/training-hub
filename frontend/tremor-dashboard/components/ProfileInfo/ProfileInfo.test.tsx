import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

// Components
import ProfileInfo from "./ProfileInfo";

// Constants
import { AVATAR_IMAGE } from "@/constants";

// Mocking usePathname hook
jest.mock("next/navigation", () => ({
  usePathname: jest
    .fn()
    .mockReturnValueOnce("/projects")
    .mockReturnValue("/products"),
}));

describe("Testing profile info component", () => {
  const propsDefault = {
    name: "Richard Davis",
    role: "CEO / Co-Founder",
    avatarUrl: AVATAR_IMAGE.LG,
  };

  const propsIncludeIsProject = {
    name: "Richard Davis",
    role: "CEO / Co-Founder",
    avatarUrl: AVATAR_IMAGE.LG,
    isProject: true,
  };

  it("Should match snapshot pathname is project", () => {
    const component = render(<ProfileInfo {...propsDefault} />);
    expect(component).toMatchSnapshot();
  });

  it("Should match snapshot another pathname", () => {
    const component = render(<ProfileInfo {...propsDefault} />);
    expect(component).toMatchSnapshot();
  });

  it("Should match snapshot with prop isProject is true", () => {
    const component = render(<ProfileInfo {...propsIncludeIsProject} />);
    expect(component).toMatchSnapshot();
  });
});
