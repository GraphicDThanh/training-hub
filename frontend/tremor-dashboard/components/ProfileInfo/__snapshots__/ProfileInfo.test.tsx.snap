// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing profile info component Should match snapshot another pathname 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex flex-col lg:flex-row lg:items-center justify-between"
      >
        <div
          class="flex flex-col xs:flex-row xs:items-center flex-wrap profile-info mb-3 -mt-3 -ml-2"
        >
          <div
            after-dynamic-value="Richard Davis avatar"
            class="inline-flex shadow-box-avatar rounded-full justify-center items-center overflow-hidden shadow-lg w-[50px] h-[50px]"
          >
            <img
              alt="Richard Davis avatar"
              class="object-cover shadow-lg w-[50px] h-[50px]"
              data-nimg="1"
              decoding="async"
              fetchpriority="high"
              height="50"
              src="http://localhost/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=64&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75 2x"
              style="color: transparent;"
              width="50"
            />
          </div>
          <div
            class="py-4 ml-6"
            data-testid="profile-info-lg"
          >
            <h2
              class="font-semibold text-xl text-primary dark:text-dark-primary capitalize"
            >
              Richard Davis
            </h2>
            <p
              class="text-tremor-default font-normal text-tertiary dark:text-dark-romance"
            >
              CEO / Co-Founder
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex flex-col lg:flex-row lg:items-center justify-between"
    >
      <div
        class="flex flex-col xs:flex-row xs:items-center flex-wrap profile-info mb-3 -mt-3 -ml-2"
      >
        <div
          after-dynamic-value="Richard Davis avatar"
          class="inline-flex shadow-box-avatar rounded-full justify-center items-center overflow-hidden shadow-lg w-[50px] h-[50px]"
        >
          <img
            alt="Richard Davis avatar"
            class="object-cover shadow-lg w-[50px] h-[50px]"
            data-nimg="1"
            decoding="async"
            fetchpriority="high"
            height="50"
            src="http://localhost/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75"
            srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=64&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75 2x"
            style="color: transparent;"
            width="50"
          />
        </div>
        <div
          class="py-4 ml-6"
          data-testid="profile-info-lg"
        >
          <h2
            class="font-semibold text-xl text-primary dark:text-dark-primary capitalize"
          >
            Richard Davis
          </h2>
          <p
            class="text-tremor-default font-normal text-tertiary dark:text-dark-romance"
          >
            CEO / Co-Founder
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Testing profile info component Should match snapshot pathname is project 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex flex-col lg:flex-row lg:items-center justify-between"
      >
        <div
          class="flex flex-col xs:flex-row xs:items-center flex-wrap profile-info mb-3 -mt-3 -ml-2"
        >
          <div
            after-dynamic-value="Richard Davis avatar"
            class="inline-flex shadow-box-avatar rounded-full justify-center items-center overflow-hidden shadow-lg w-[50px] h-[50px]"
          >
            <img
              alt="Richard Davis avatar"
              class="object-cover shadow-lg w-[50px] h-[50px]"
              data-nimg="1"
              decoding="async"
              fetchpriority="high"
              height="50"
              src="http://localhost/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=64&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75 2x"
              style="color: transparent;"
              width="50"
            />
          </div>
          <div
            class="py-4 ml-6"
            data-testid="profile-info-lg"
          >
            <h2
              class="font-semibold text-xl text-primary dark:text-dark-primary capitalize"
            >
              Richard Davis
            </h2>
            <p
              class="text-tremor-default font-normal text-tertiary dark:text-dark-romance"
            >
              CEO / Co-Founder
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex flex-col lg:flex-row lg:items-center justify-between"
    >
      <div
        class="flex flex-col xs:flex-row xs:items-center flex-wrap profile-info mb-3 -mt-3 -ml-2"
      >
        <div
          after-dynamic-value="Richard Davis avatar"
          class="inline-flex shadow-box-avatar rounded-full justify-center items-center overflow-hidden shadow-lg w-[50px] h-[50px]"
        >
          <img
            alt="Richard Davis avatar"
            class="object-cover shadow-lg w-[50px] h-[50px]"
            data-nimg="1"
            decoding="async"
            fetchpriority="high"
            height="50"
            src="http://localhost/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75"
            srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=64&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75 2x"
            style="color: transparent;"
            width="50"
          />
        </div>
        <div
          class="py-4 ml-6"
          data-testid="profile-info-lg"
        >
          <h2
            class="font-semibold text-xl text-primary dark:text-dark-primary capitalize"
          >
            Richard Davis
          </h2>
          <p
            class="text-tremor-default font-normal text-tertiary dark:text-dark-romance"
          >
            CEO / Co-Founder
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Testing profile info component Should match snapshot with prop isProject is true 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex flex-col lg:flex-row lg:items-center justify-between"
      >
        <div
          class="flex flex-col xs:flex-row xs:items-center flex-wrap profile-info my-1"
        >
          <div
            after-dynamic-value="Richard Davis avatar"
            class="inline-flex shadow-box-avatar rounded-full justify-center items-center overflow-hidden shadow-lg w-[50px] h-[50px]"
          >
            <img
              alt="Richard Davis avatar"
              class="object-cover shadow-lg w-[50px] h-[50px]"
              data-nimg="1"
              decoding="async"
              fetchpriority="high"
              height="50"
              src="http://localhost/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=64&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75 2x"
              style="color: transparent;"
              width="50"
            />
          </div>
          <div
            class="py-4 ml-6"
            data-testid="profile-info-lg"
          >
            <h2
              class="font-semibold text-xl text-primary dark:text-dark-primary capitalize"
            >
              Richard Davis
            </h2>
            <p
              class="text-tremor-default font-normal text-tertiary dark:text-dark-romance"
            >
              CEO / Co-Founder
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex flex-col lg:flex-row lg:items-center justify-between"
    >
      <div
        class="flex flex-col xs:flex-row xs:items-center flex-wrap profile-info my-1"
      >
        <div
          after-dynamic-value="Richard Davis avatar"
          class="inline-flex shadow-box-avatar rounded-full justify-center items-center overflow-hidden shadow-lg w-[50px] h-[50px]"
        >
          <img
            alt="Richard Davis avatar"
            class="object-cover shadow-lg w-[50px] h-[50px]"
            data-nimg="1"
            decoding="async"
            fetchpriority="high"
            height="50"
            src="http://localhost/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75"
            srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=64&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=128&q=75 2x"
            style="color: transparent;"
            width="50"
          />
        </div>
        <div
          class="py-4 ml-6"
          data-testid="profile-info-lg"
        >
          <h2
            class="font-semibold text-xl text-primary dark:text-dark-primary capitalize"
          >
            Richard Davis
          </h2>
          <p
            class="text-tremor-default font-normal text-tertiary dark:text-dark-romance"
          >
            CEO / Co-Founder
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
